demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/01/Version20240108110342.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20240108110342 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'refs T34551: alters the property customer for the procedure table
+            This property was used to indicate a blueprint that is just for a specific customer
+            Now this property shall indicate that the procedure is owned by an organisation within this customer
+            This is necessary to determine all customer related activity.
+            The logic to filter a customer-blueprint has to be adjusted to check the additional property _p_master
```

Comment:
Note to self for the review: Check this one that it does not break the "usual" orga procedure template logic